sdk/core/azure-core-tracing-opentelemetry/test/ut/azure_core_otel_test.cpp

```diff
@@ -542,15 +558,15 @@ TEST_F(OpenTelemetryTests, SetStatus)
 
   // Set to Unset.
   {
-    std::shared_ptr<Azure::Core::Tracing::TracerProvider> traceProvider
-        = std::make_shared<Azure::Core::Tracing::OpenTelemetry::OpenTelemetryProvider>(
-            CreateOpenTelemetryProvider());
+    auto traceProvider = Azure::Core::Tracing::OpenTelemetry::OpenTelemetryProvider::Create(
+        CreateOpenTelemetryProvider());
 
-    auto tracer = traceProvider->CreateTracer("TracerName");
-    auto span = tracer->CreateSpan("StatusSpan");
+    auto tracer = Azure::Core::Tracing::_internal::TracerImplFromTracer(traceProvider)
+                      ->CreateTracer("TracerName", "");
+    auto span = tracer->CreateSpan("StatusSpan", {});
     EXPECT_TRUE(span);
 
-    span->SetStatus(Azure::Core::Tracing::_internal::SpanStatus::Unset);
+    span->SetStatus(Azure::Core::Tracing::_internal::SpanStatus::Unset, "");
```

Comment:
Super nit, but I think `{}` is better than `""` (there is more than one occurrence), because it does not require to statically allocate `[ ``\0`` ]` and to call `std::string(const char*)`.

Reply:
Ah, sorry, nevermind, I did not notice this was in test code.